docs/custom_mutators.md

```diff
@@ -48,6 +48,7 @@ C/C++:
 ```c
 void *afl_custom_init(afl_state_t *afl, unsigned int seed);
 unsigned int afl_custom_fuzz_count(void *data, const unsigned char *buf, size_t buf_size);
+void afl_custom_splice_optout(void *data);
```

Comment:
Rust implementation?

Reply:
uh can you do that? :) not sure what has to be done. pretty sure afl_custom_send is missing too